src/main/java/frc/robot/Controls.java

```diff
@@ -134,11 +144,19 @@ private void configureDrivebaseBindings() {
                       .withRotationalRate(newSpeeds.omegaRadiansPerSecond);
                 })
             .withName("Drive"));
-    s.drivebaseSubsystem
-        .applyRequest(() -> brake)
-        .ignoringDisable(true)
-        .withName("Brake")
-        .schedule();
+    RobotModeTriggers.disabled()
+        .onTrue(
+            s.drivebaseSubsystem
+                .applyRequest(() -> brake)
+                .ignoringDisable(true)
+                .withName("Disable Brake"));
+    if (DriverStation.isDisabled()) {
+      s.drivebaseSubsystem
+          .applyRequest(() -> brake)
+          .ignoringDisable(true)
+          .withName("Brake")
+          .schedule();
```

Comment:
Similar to above, I think this won't do anything until the robot becomes enabled since motors can't be actuated during disabled mode, so the wheels won't move to the X position until the robot is enabled (and assuming nothing else overrides the swerve command set here before the robot is enabled). That generally isn't a good thing, as when the robot is next enabled the wheels will immediately rotate out of X to whatever direction the robot wants to go, and that initial rotation out of X adds noise to the swerve odometry (and a delay in movement while the robot waits for the wheels to rotate before the drive motors start spinning.